### PR TITLE
Add documentation to compile and use WarpX on Adastra (CINES) supercomputer

### PR DESCRIPTION
**Pull request by lucafedeli88**
This PR adds the documentation to compile and use WarpX on Adastra (CINES) supercomputer

### CODE REVIEW

**Comment by ax3l** [comment]
@lucafedeli88 Awesome, thank you!

For the page to show up, please link it in the table of contents tree by adding the new file in `Docs/source/install/hpc.rst`

**Comment by lucafedeli88** [comment]
> Docs/source/install/hpc.rst

ah, right! I always forget to do this!

File: Docs/source/install/hpc/adastra.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+.. code-block:: bash
+
+   source $HOME/adastra_warpx.profile
+
+And since Adastra does not yet provide a module for them, install c-blosc and ADIOS2:
+
+.. code-block:: bash
+
+   export CMAKE_PREFIX_PATH=${HOME}/sw/adastra/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
+   export CMAKE_PREFIX_PATH=${HOME}/sw/adastra/gpu//adios2-2.8.3:$CMAKE_PREFIX_PATH
```
Comment (ax3l) [inline Docs/source/install/hpc/adastra.rst]:
```suggestion
   export CMAKE_PREFIX_PATH=${HOME}/sw/adastra/gpu/adios2-2.8.3:$CMAKE_PREFIX_PATH
```

File: Docs/source/install/hpc/adastra.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+.. code-block:: bash
+
+   source $HOME/adastra_warpx.profile
+
+And since Adastra does not yet provide a module for them, install c-blosc and ADIOS2:
+
+.. code-block:: bash
+
+   export CMAKE_PREFIX_PATH=${HOME}/sw/adastra/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
```
Comment (ax3l) [inline Docs/source/install/hpc/adastra.rst:54]: Is the default home quota large enough? I started transitioning on NERSC (10GB default) now to other locations.

Here, you could use `$WORKDIR/sw` or `$SHAREDWORKDIR/$USER/sw` it seems.

**Review by ax3l — APPROVED** [review]
LGTM, thanks a lot! :tada: 

Waiting for Neil to test and confirm.

File: Docs/source/install/hpc/adastra.rst (hunk 2 above)
Comment (ax3l) [reply Docs/source/install/hpc/adastra.rst:54]: Ping @lucafedeli88

Comment (ax3l) [reply Docs/source/install/hpc/adastra.rst:54]: ping @NeilZaim @lucafedeli88 

can be a follow-up :)

**Pull request merged**